src/activity_actor.cpp

```diff
@@ -349,7 +349,7 @@ bool aim_activity_actor::load_RAS_weapon()
 
     // Burn 0.2% max base stamina x the strength required to fire.
     you.mod_stamina( gun->get_min_str() * static_cast<int>( 0.002f *
-                     get_option<int>( "PLAYER_MAX_STAMINA" ) ) );
+                     you.get_stamina_max() ) );
```

Comment:
I think this one should be "PLAYER_MAX_STAMINA_BASE" without cardio modifier, otherwise being more fit make you consume more stamina.

Reply:
Hmm, probably neither... It probably needs to make a baseline assumption of what your stamina should be at zero. Fortunately that's easy to make.